content/docs/pipelines/pipelines-quickstart.md

```diff
@@ -25,6 +25,8 @@ Follow these steps to deploy Kubeflow and open the pipelines dashboard:
     alt="Kubeflow UI"
     class="mt-3 mb-3 border border-info rounded">
 
+   If IAP is skipped, run ```kubectl port-forward -n kubeflow `kubectl get pods -n kubeflow --selector=service=ambassador -o jsonpath='{.items[0].metadata.name}'` 8080:80``` and go to http://localhost:8080/
```

Comment:
For clarity, change the following:

If IAP is skipped

To this:

If you skipped the IAP option when deploying Kubeflow